src/typechecker/FStar.TypeChecker.Cfg.fsti

```diff
@@ -93,8 +93,8 @@ type primitive_step = {
     auto_reflect:option int;
     strong_reduction_ok:bool;
     requires_binder_substitution:bool;
-    interpretation:(psc -> FStar.Syntax.Embeddings.norm_cb -> args -> option term);
-    interpretation_nbe:(NBE.nbe_cbs -> NBE.args -> option NBE.t)
+    interpretation:(psc -> FStar.Syntax.Embeddings.norm_cb -> universes -> args -> option term);
+    interpretation_nbe:(NBE.nbe_cbs -> universes -> NBE.args -> option NBE.t)
```

Comment:
I added the universes to the interpretation functions, so that if `op u#us v1 ... vn` is reduced, the interpretation has access to the universes `u#us`, e.g., in case it needs to rebuild a term that mentions those universes.